### PR TITLE
create pull

### PR DESCRIPTION
**Pull request by SoundhararajanL**
-create pull request

### CODE REVIEW

**Comment by SoundhararajanL** [comment]
created pull

File: Tamilselvan/tamil.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("")
```
Comment (SoundhararajanL) [inline Tamilselvan/tamil.py]: please add ur name print("here")

**Review by SoundhararajanL — COMMENTED** [review]
code review by soundhar

File: Tamilselvan/tamil.py (hunk 1 above)
Comment (tamilasm9095) [reply Tamilselvan/tamil.py]: ok will do i and let you know

**Comment by tamilasm9095** [comment]
name has been added

File: soundhar/soundhar.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print("soundhar")
```
Comment (tamilasm9095) [inline soundhar/soundhar.py]: please add full name

**Review by tamilasm9095 — COMMENTED** [review]
code review by tamil

File: Tamilselvan/tamil.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+print("NAME : Tamilselvan")
+# added new line
```
Comment (SoundhararajanL) [inline Tamilselvan/tamil.py]: please add new line here

Comment (tamilasm9095) [reply Tamilselvan/tamil.py]: new line has been added

File: soundhar/soundhar.py [hunk 4]
```diff
@@ -0,0 +1,2 @@
+print("soundhararajan L ")
+print("new line")
```
Comment (tamilasm9095) [inline soundhar/soundhar.py:2]: add new cmt line here

Comment (tamilasm9095) [inline soundhar/soundhar.py]: plaese add cmt line

File: Tamilselvan/tamil.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+# new line with emp no 77324
```
Comment (SoundaryaNagaraj) [inline Tamilselvan/tamil.py]: add new print line

**Review by tamilasm9095 — APPROVED** [review]
add new

File: soundhar/soundhar.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
+# cmt line added
```
Comment (SoundaryaNagaraj) [inline soundhar/soundhar.py:3]: please add any other line

File: soundarya/soundarya.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+print("hello")
+# tamilselvan added 
+# after
```
Comment (SoundhararajanL) [inline soundarya/soundarya.py:3]: add another print line

File: Tamilselvan/tamil.py [hunk 8]
```diff
@@ -0,0 +1,3 @@
+print("NAME : Tamilselvan")
+# added new line
+# new line with emp no 744
```
Comment (SoundhararajanL) [inline Tamilselvan/tamil.py:3]: remove all cmt lines

File: Tamilselvan/tamil.py [hunk 9]
```diff
@@ -0,0 +1,2 @@
+print("NAME : Tamilselvan")
+
```
Comment (SoundhararajanL) [inline Tamilselvan/tamil.py:2]: please add new line here

**Pull request merged**